src/assets/wise5/authoringTool/teacher-node-icon/teacher-node-icon.component.ts

```diff
@@ -3,18 +3,36 @@ import { ProjectService } from '../../services/projectService';
 import { NodeIconComponent } from '../../vle/node-icon/node-icon.component';
 import { NodeIconChooserDialog } from '../../common/node-icon-chooser-dialog/node-icon-chooser-dialog.component';
 import { Component } from '@angular/core';
+import { MatButtonModule } from '@angular/material/button';
+import { FlexLayoutModule } from '@angular/flex-layout';
+import { CommonModule } from '@angular/common';
+import { MatTooltipModule } from '@angular/material/tooltip';
+import { MatIconModule } from '@angular/material/icon';
+import { MatBadgeModule } from '@angular/material/badge';
 
 @Component({
+  imports: [
+    CommonModule,
+    FlexLayoutModule,
+    MatBadgeModule,
+    MatButtonModule,
+    MatIconModule,
+    MatTooltipModule
+  ],
   selector: 'teacher-node-icon',
-  templateUrl: '../../vle/node-icon/node-icon.component.html',
-  styleUrls: ['../../vle/node-icon/node-icon.component.scss']
+  standalone: true,
+  styleUrl: '../../vle/node-icon/node-icon.component.scss',
+  templateUrl: '../../vle/node-icon/node-icon.component.html'
 })
 export class TeacherNodeIconComponent extends NodeIconComponent {
-  constructor(protected dialog: MatDialog, protected projectService: ProjectService) {
+  constructor(
+    protected dialog: MatDialog,
+    protected projectService: ProjectService
+  ) {
```

Comment:
Can you put this in one line?

Reply:
It looks like prettier automatically puts the constructor parameters on separate lines if any of them have a protected or private modifier. I couldn't figure out how to prevent this. It might be a bug introduced in the newer prettier version we're using.